langfuse/serializer.py

```diff
@@ -31,6 +31,9 @@ def default(self, obj: Any):
             # Timezone-awareness check
             return serialize_datetime(obj)
 
+        if isinstance(obj, Exception):
```

Comment:
I suggest changing it to this, WDYT?
```
if isinstance(obj, (Exception, KeyboardInterrupt)):
    return f"{type(obj).__name__}: {str(obj)}"
```

Reply:
That makes the code stronger, i will change it.